README.md

```diff
@@ -1 +1,411 @@
+az ad app permission grant --id ${AZURE_CLIENT_ID} --api 00000002-0000-0000-c000-000000000000 --expires never
+```
+
+You might see an error similar to the following, but that is OK, the permissions should have gone through still:
+
+_Operation failed with status: 'Conflict'. Details: 409 Client Error: Conflict for url: https://graph.windows.net/e7985bc4-a3b3-4f37-b9d2-fa256023b1ae/oauth2PermissionGrants?api-version=1.6_
+
+Finally, you need to grant admin permissions on the Azure Active Directory to the service principal because it will need to create other service principals for your AKSCluster:
+
+```console
+# grant admin consent to the service princinpal you created
+az ad app permission admin-consent --id "${AZURE_CLIENT_ID}"
+```
+
+_Note: You might need Global Administrator role to Grant admin consent for Default Directory. Please contact the administrator of your Azure subscription. To check your role, go to Azure Active Directory -> Roles and administrators. You can find your role(s) by clicking on Your Role (Preview)_
+
+After these steps are completed, you should have the following file on your local filesystem:
+
+- crossplane-azure-provider-key.json
+
+#### Setup Azure ProviderConfig
+
+Before creating any resources, we need to create and configure an Azure cloud provider resource in Crossplane, which stores the cloud account information in it. All the requests from Crossplane to Azure Cloud will use the credentials attached to this provider resource. The following command assumes that you have a crossplane-azure-provider-key.json file that belongs to the account you’d like Crossplane to use.
+
+Now we’ll create our Secret that contains the credential and ProviderConfig resource that refers to that secret:
+
+```console
+kubectl create secret generic azure-account-creds -n upbound-system --from-file=credentials=./crossplane-azure-provider-key.json
+```
+
+```console
+cat > provider.yaml <<EOF
+---
+apiVersion: azure.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+  namespace: upbound-system
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-account-creds
+      key: credentials
+EOF
+```
+
+Apply it to the cluster:
+```console
+kubectl apply -f "provider.yaml"
+```
+The output will look like the following:
+
+```shell
+provider.azure.crossplane.io/default created
+```
+
+Crossplane resources use the ProviderConfig named ```default``` if no specific ProviderConfig is specified, so this ProviderConfig will be the default for all Azure resources.
+
+### We are now ready to provision resources:
+
+#### Create Network Fabric
+
+The example network composition includes the creation of a Resource Group, Virtual Network, and Subnets:
+
+```console
+kubectl apply -f examples/network.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+#### Invite App Teams to you Organization in Upbound Cloud
```

Comment:
did you walk through these steps using the GUI for creating database? are the current and do they work? they may have been a legacy thing from platform-ref-aws that wanted to showcase creating resources with both kubectl and upbound cloud GUI - let's ensure they work correctly :)

Reply:
or just remove them and use kubectl only instructions for now, as discussed in https://github.com/upbound/platform-ref-azure/pull/1/files#r778387800 below

Reply:
is there any reason to keep this UI section (and the next one) at all? it seems like its a potential distraction from the kubectl workflow we have described here for all 3 main resources

Reply:
agree, I think the downside of removing it is taking away from experiencing the ui. we could remove it and then reincorporate once all of the platform refs have been streamlined with no network claim

Reply:
will take it out

Reply:
that makes a lot of sense to me! we can include an entire UI walkthrough once everything is polished there too :)